src/display/network.js

```diff
@@ -593,8 +569,6 @@ PDFNetworkStreamRangeRequestReader.prototype = {
   },
 };
 
-setPDFNetworkStreamClass(PDFNetworkStream);
```

Comment:
In [src/pdf.js](https://github.com/mukulmishra18/pdf.js/commit/712de53d0ac469348a4d05285e4be42758e5cb8f#diff-3841ca9d0e4702635b24c39b593d0610R34), we are trying to detect the environment(using `isNodeJS`), and then setting `PDFNodeStream` or `PDFNetworkStream`, see one of the TODO here: https://github.com/mozilla/pdf.js/pull/8712#pullrequestreview-52920496